test/extended/image_ecosystem/scl.go

```diff
@@ -35,6 +50,12 @@ func defineTest(image string, t tc, oc *exutil.CLI) {
 				Image: t.DockerImageReference,
 			})
 			_, err := oc.KubeClient().CoreV1().Pods(oc.Namespace()).Create(pod)
+			if err != nil {
```

Comment:
instead of here (or in addition to), add this to the global error handling for failed tests.